src/datadoc/frontend/fields/display_dataset.py

```diff
@@ -300,6 +294,27 @@ class DatasetIdentifiers(str, Enum):
         editable=True,
         id_type=DATASET_METADATA_DATE_INPUT,
     ),
+    DatasetIdentifiers.CUSTOM_TYPE: MetadataInputField(
+        identifier=DatasetIdentifiers.CUSTOM_TYPE.value,
+        display_name="Egendefinert type",
+        description="",
+        disabled=True,
+    ),
+    DatasetIdentifiers.USE_RESTRICTION: MetadataInputField(
+        identifier=DatasetIdentifiers.USE_RESTRICTION.value,
+        display_name="Brukerrestriksjon",
+        description="",
+    ),
+    DatasetIdentifiers.USE_RESTRICTION_DATE: MetadataPeriodField(
+        identifier=DatasetIdentifiers.USE_RESTRICTION_DATE.value,
+        display_name="Bruksrestriksjonsdato",
+        description="",
+    ),
+    DatasetIdentifiers.CONTAINS_PERSONAL_DATA: MetadataInputField(
```

Comment:
This should be obligatory ref: https://statistics-norway.atlassian.net/wiki/spaces/MPD/pages/3039133732/DataDoc+-+Krav+til+dokumentasjon+av+datasett+p+Dapla#inneholder-personopplysninger%2FcontainsPersonalData

Also this value is boolean so we can use a checkbox for it.